.env

```diff
@@ -2,4 +2,5 @@
 BUILD_VARIANT=rcvr
 NEXT_PUBLIC_BUILD_VARIANT=$BUILD_VARIANT
 
-NEXT_PUBLIC_API_BASE=https://api.rcvr.app/
\ No newline at end of file
+NEXT_PUBLIC_API_BASE=https://api.rcvr.app/
+NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY=pk_test_Q2N3ljTzzBgxO4uOlsv5BS7u00f3eocyfC
```

Comment:
```suggestion
NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY=pk_live_4e9OH54b1NzmfES809cLedMu00F8m8gRka
```

Reply:
Can we still use the test key for testing? Or is it invalid now?

Reply:
We'll leave this here so we can use the test key in development